### ISSUE TITLE
Error in load(): unknown type 240 in IRC model.R

### ISSUE THREAD

**Issue by vlntnzl** [EricPetermann/radon_exposure_germany#3]
Error message when running model.qmd:

```
Quitting from lines 9-400 [unnamed-chunk-1] (model.qmd)                                                   
Error in `load()`:                                                                           
! ReadItem: unknown type 240, perhaps written by later version of R                           
Backtrace:                                                                                    
 1. base::load("IRC model.R")                                                                             
Execution halted 
```

@EricPetermann Have you ever seen this error and have any idea what is going wrong?

**Comment by vlntnzl** [comment]
A similar problem with different solutions is discussed [here](https://stackoverflow.com/a/49287210). Is it necessary to load the IRC model.R?

**Comment by vlntnzl** [comment]
Another possibility: [File corruption](https://stat.ethz.ch/pipermail/r-help/2015-March/426562.html). Maybe during compression of the file. Or: Loading this binary file from Windows OS in Linux OS.

**Comment by vlntnzl** [comment]
A new error message appears after a direct upload without compression:

```
Quitting from lines 9-400 [unnamed-chunk-1] (model.qmd)
Error in `readChar()`:
! cannot open the connection
Backtrace:
 1. base::load("IRC model.R")
 2. base::readChar(con, 5L, useBytes = TRUE)
Execution halted

```

Error messages are solved [here](https://www.programmingr.com/r-error-messages/readcharcon-5l-usebytes-true-cannot-open-the-connection/), [here](https://stackoverflow.com/a/11516995) and [here](https://www.positioniseverything.net/error-in-readcharcon-5l-usebytes-true-cannot-open-the-connection/).

The problem here was an incorrect path. It should actually have been `/app/data/`, but it became `/app/data/data/` due to a copy operation and so quarto could not find the directory. 

Now we are back to the original error message.

**Comment by vlntnzl** [comment]
Same error with R-Base 4.2.2 (not latest):

```
Quitting from lines 9-402 [unnamed-chunk-1] (model.qmd)
Error in `load()`:
! ReadItem: unknown type 240, perhaps written by later version of R
Backtrace:
 1. base::load("IRCmodel.R")
Execution halted
```

It seems necessary to create the model under Linux.

**Comment by vlntnzl** [comment]
The solution was actually to create the IRC model on the target system.

Fixed in commit: 5c281b7
